listenbrainz/webserver/static/js/src/listens/ListenControl.tsx

```diff
@@ -43,9 +44,14 @@ const ListenControl = (props: ListenControlProps) => {
     // When using the link property,
     // render an anchor tag with an href instead of onClick
     return (
-      <a href={link} title={title} {...anchorTagAttributes}>
+      <a
+        href={link}
+        aria-label={ariaLabel ?? text}
```

Comment:
We can even do this to make sure there is something, anything, in the aria label.
Same for below in the button.

And then we don't even need the `aria-label="Open in MusicBrainz"` anymore

```suggestion
        aria-label={ariaLabel ?? title ?? text}
```